src/main/java/net/minecraftforge/fml/common/ILanguageAdapter.java

```diff
@@ -166,7 +176,8 @@ public void setInternalProxies(ModContainer mod, Side side, ClassLoader loader)
 
                             setProxy(target, proxyTarget, proxy);
                         }
-                        catch (Exception e) {
+                        catch (Exception e)
+                        {
```

Comment:
Its cpw's code, he doesn't follow the standards for reasons, and it's not really on you as an external person to fix codestyle in the project. It clutters the meat of PRs.
If I ever care, and have to work on this class myself I go in and manually format it as i'm doing it.